docs/content/docs/libs/cep.md

```diff
@@ -653,7 +653,7 @@ Pattern<Event, ?> strict = start.next("middle").where(...);
 Pattern<Event, ?> relaxed = start.followedBy("middle").where(...);
 
 // non-deterministic relaxed contiguity
-Pattern<Event, ?> nonDetermin = start.followedByAny("middle").where(...);
+Pattern<Event, ?> nonDetermine = start.followedByAny("middle").where(...);
```

Comment:
This isn't really a typo; its ust an abbreviation for nonDeterministic

Reply:
Thanks zentol. I have just rolled back those commits in cep.md.